packages/elements/src/containers/Provider.tsx

```diff
@@ -1,15 +1,9 @@
 import * as React from 'react';
 
 import { MockUrlResult } from '../components/TryIt/mocking-utils';
-import { IActiveInfo, NodeIconMapping } from '../types';
+import { IActiveInfo } from '../types';
 
-export interface IProvider extends IActiveInfo {
-  authToken?: string;
-  isStoplightProjectComponent?: boolean;
-}
-
-const defaultIcons: NodeIconMapping = {};
-export const IconsContext = createNamedContext<NodeIconMapping>('IconsContext', defaultIcons);
```

Comment:
Where was this used before? Elements seem to be working fine without it

Reply:
That's the thing - I haven't found a single usage of that :joy: